05-single-hop-route/task02.py

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env python3
+import sys
+import os
+import argparse
+sys.path.append("../testutils")
+
+from iotlab import IoTLABNode, IoTLABExperiment  # noqa: E402
+from common import SingleHopNode, single_hop_run, print_results  # noqa: E402
+
+if __name__ == "__main__":
+    p = argparse.ArgumentParser()
+    p.add_argument('riotbase', nargs='?',
+                   help='Location of RIOT folder')
+    args = p.parse_args()
+    riotbase = args.riotbase
+
+    if not riotbase:
+        p.print_help()
+        sys.exit(1)
+
+    os.chdir(os.path.join(riotbase, "tests/gnrc_udp"))
+
+    # Create IoTLAB experiment
+    exp = IoTLABExperiment("RIOT-release-test-05-02",
+                           [IoTLABNode(extra_modules=["gnrc_pktbuf_cmd"]),
+                            IoTLABNode(extra_modules=["gnrc_pktbuf_cmd"])])
+
+    N = 10
+
+    try:
+        addr = exp.nodes_addresses
+        iotlab_cmd = "make IOTLAB_NODE={} BOARD=iotlab-m3 term"
+        source = SingleHopNode(iotlab_cmd.format(addr[0]))
+        dest = SingleHopNode(iotlab_cmd.format(addr[1]))
+        results = []
+
+        ip_src = "affe::1/120"
+        ip_dest = "beef::1/64"
+        src_route = "::"
+        dest_route = "::"
+        disable_rdv = False
+        count = 100
+        tolerance = 10
+
+        for i in range(N):
+            source.reboot()
+            dest.reboot()
+
+            packet_loss, buf_source, buf_dest = single_hop_run(
+                    source, dest, ip_src, ip_dest, src_route,
+                    dest_route, disable_rdv, count)
+
+            results.append([packet_loss, buf_source, buf_dest])
+
+            assert(packet_loss < tolerance)
```

Comment:
When running this task I get the following message:
```
'<' not supported between instances of 'NoneType' and 'int'
Test failed!
```
Apparently `packet_loss` is None.

Reply:
Most likely because [this line](https://github.com/RIOT-OS/Release-Specs/blob/8aface590373dc583af7053416e9f86aac2333c8/testutils/mixins.py#L40) is never reached.

Reply:
yes, packet_losses shouldn't be `None` but 0

Reply:
If packet loss is a percentage, it should rather be 100 as a failure case ;)